spec/models/collection_spec.rb

```diff
@@ -82,6 +82,12 @@
       expect(collection.errors.full_messages).to \
         include /Sorry, a collection can only have 10 tags./
     end
+
+    it "does not raise an 500 error when header_image_url is not a valid URL" do
```

Comment:
This test does not test the desired behaviour, which is that this will result in a validation error. Generally this test is a bit in the wrong place because saving a model can never result in an error 500 - error codes are generated int the controller, not the model.